src/middlewares/error-handler.ts

```diff
@@ -0,0 +1,8 @@
+import { NextFunction, Request, Response } from 'express';
+export const ErrorHandler = (err: any, req: Request, res: Response, next: NextFunction) => {
+  if (!err) next();
+  res.status(500).json({
```

Comment:
```suggestion
  if (!err) next();
  
  res.status(500).json({
```

Ещё пустую строку забыл. Логические элементы между собой разделяются пустой строкой